assets/javascript/petapp.js

```diff
@@ -276,7 +271,7 @@ $(document).ready(function() {
   // this retrieves business data
   function getBizData() {
     var search = $("#selection-input").val();
-    var location = $("#address-input").val();
+    var location = $("#pac-input").val();
```

Comment:
Hey, Mark - is there a reason you wanted to revert this to #pac-input? "address-input" seemed more easily understandable to me... but maybe just a matter of when you last pulled :)